Tween/CustomAnimation.cs

```diff
@@ -28,33 +28,67 @@ public class CustomAnimation : TimedAnimation
 		/// <param name="to">To value</param>
 		/// <param name="duration">Duration of the animation</param>
 		/// <param name="easingFunction">The easing function for the interpolation</param>
-		public CustomAnimation(Action<int> customUpdate, int from, int to, float duration = 1.0f, Func<float, float> easingFunction = null)
-			: this((float value) => customUpdate((int)value), from, to, duration, easingFunction) {}
+		public CustomAnimation(Action<float> customUpdate, float from = 0, float to = 1.0f, float duration = 1.0f, Func<float, float> easingFunction = null)
+			: base((GameObject)null, duration, easingFunction)
+		{
+			if (customUpdate == null)
+			{
+				throw new ArgumentNullException("customUpdate");
```

Comment:
There's probably a reason for this in the wider tween library I can't see in this PR, but why do we throw when the custom update is null but abort with a log in other animation types when the target is null?

Reply:
But `TimedAnimation` and `AnimationCollection`s don't as far as I can see? `DelegateAnimation` does though. I guess it's not a problem for this PR but it's weird since animations are often created when needed, not always cached early. One of these ways will throw and one will just log an error and continue.

Reply:
Anything invalid that you can catch on input to a function or constructor, it should be a throw. Be explicit. If you don't want them to put null it don't let them, if you don't want a negative number, don't let them. If it's missing in other places then it's something we need to improve on, but as you say it's beyond the scope of this PR

Reply:
I agree with you but it is a bit less clear cut in this case what we should do in the constructor, because of .Play preventing exceptions